webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -278,10 +278,11 @@ Kommunicate.richMsgEventHandler = {
 
         var target = e.target || e.srcElement;
         var requestType = target.dataset.requesttype;
-        var replyText = target.title || target.innerHTML;
-        var buttonType = target.dataset.buttontype;
+        var buttonType = target.dataset.buttontype || target.type;
         var data = {};
-        var form =target.parentElement.getElementsByClassName('km-btn-hidden-form')[0];
+        var form =target.parentElement.getElementsByClassName('km-btn-hidden-form')[0]
+        var isActionableForm = (form.id == "mck-actionable-form");
```

Comment:
@jithinbalu will it be only one form? what if there are multiple forms in the chat history?

Reply:
@devashishmamgain  the form object is created from parentElement of the clicked button (line 283)

Reply:
@jithinbalu so at any given point of time, it will be always 1 element with id = "mck-actionable-form"?

Reply:
@devashishmamgain ok got the point there should not be multiple forms with the same id. i will improve it

Reply:
@devashishmamgain "id" is removed from the form and id depended code also improved